src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -515,6 +515,18 @@ public ToolTipDescription ToolTipDescriptionFor(int index)
          };
       }
 
+      private void setUnitAndDimension(ColumnMappingDTO model)
+      {
+         var supportedDimensions = _columnInfos.First(x => x.DisplayName == model.MappingName).SupportedDimensions;
```

Comment:
I feel that I have seens this code
`_columnInfos.First(x => x.DisplayName == model.MappingName` e.g. finding a column by display name so many times, that I am believe it needs to be refactored.
What about you create  cache with DisplayName keys instead of IREadOnlyList. That way you can simply do columnInfos[model.MappingName].

Reply:
Cache is a ienumerable by default. It will be the same code everywhere except where you can simplify it drastically